content/kubeone/master/tutorials/creating_clusters_baremetal/_index.en.md

```diff
@@ -3,7 +3,328 @@ title = "Creating a Kubernetes Cluster on Bare-metal"
+
+\* containerd will be installed by default for newly-created Kubernetes 1.21+
+clusters
+
+It's possible to configure which components are installed and how they are
+configured by adjusting the KubeOne configuration manifest that we'll create
+later in the Step 3 (Provisioning The Cluster). To see possible configuration
+options, refer to the configuration manifest reference which can be obtained
+by running `kubeone config print --full`.
+
+## Step 1 — Downloading KubeOne
+
+The easiest way to download KubeOne is to use our installation script.
+The following command will download and run the script:
+
+```shell
+curl -sfL get.kubeone.io | sh
+```
+
+The script downloads the latest version of KubeOne from GitHub, and unpacks it
+in the `/usr/local/bin` directory. Additionally, the script unpacks the example
+Terraform configs, addons, and helper scripts in your current working
+directory. At the end of the script output, you can find the path to the
+unpacked files:
+
+{{% notice note %}}
+The addons and helper scripts are supposed to be used for advanced deployments
+and we will not use those in this tutorial. They're not required by KubeOne, so
+you're not required to keep to them.
+{{% /notice %}}
+
+```shell
+...
+Kubermatic KubeOne has been installed into /usr/local/bin/kubeone
+Terraform example configs, addons, and helper scripts have been downloaded into the ./kubeone_1.2.0-beta.1_linux_amd64 directory
+```
+
+You can confirm that KubeOne has been installed successfully by running the
+`kubeone version` command. If you see an error, ensure that `/usr/local/bin` is
+in your `PATH` or modify the script to install KubeOne in a different place.
+You can also check the [Getting KubeOne guide][getting-kubeone] for alternative
+installation methods.
+
+## Step 2 — Creating The Infrastructure
+
+With KubeOne installed and the environment configured, we're ready to create the infrastructure for our cluster.
+Because we are not using any provider, it is up to you, to create the necessary infrastructure.
+Please refer to the [Infrastructure Management][infrastructure-management] for the requirements.
+You need to provide Infrastructure for both the control-plane and the worker nodes.
+
+
+Usually the Kubermatic machine-controller would take care of the worker node management.
+However, without a provider we can not make use of it.
+
+For the following steps we assume that the required infrastructure is in place and ssh access is ensured.
+More information about ssh requirements and configuration can be found in the [Configuring SSH][configuring-ssh] guide.
+
+## Step 3 — Provisioning The Cluster
+
+Now that we have the infrastructure, we can use KubeOne to provision a Kubernetes cluster.
+
+The first step is to create a KubeOne configuration manifest that describes how
+the cluster will be provisioned, which Kubernetes version will be used,
+and more. To see possible configuration options reference, you can run `kubeone config print --full`.
```

Comment:
The most important part of the configuration manifest for bare metal setups is the list of instances that will be used. I'd say something along:

```suggestion
The first step is to create a KubeOne configuration manifest that contains a list of instances that will be used, and that describes how the cluster will be provisioned, which Kubernetes version will be used,
and more. To see possible configuration options reference, you can run `kubeone config print --full`.
```